src/pages/components/button/accessibility.mdx

```diff
@@ -6,314 +6,78 @@ description:
-  StructuredListCell,
-  OrderedList,
-  ListItem,
-} from 'carbon-components-react';
+<PageDescription>
+
+Design annotations are needed for specific instances shown below, but for the standard button component, Carbon already incorporates accessibility.
+
+</PageDescription>
 
 <AnchorLinks>
-  <AnchorLink>How it works</AnchorLink>
-  <AnchorLink>Accessibility considerations</AnchorLink>
-  <AnchorLink>Resources</AnchorLink>
-  <AnchorLink>Accessibility testing</AnchorLink>
+  <AnchorLink>What Carbon provides</AnchorLink>
+  <AnchorLink>Design recommendations</AnchorLink>
+  <AnchorLink>Development considerations</AnchorLink>
 </AnchorLinks>
 
-## How it works
+## What Carbon provides
 
-The `Button` component is used to trigger an action or event, such as submitting
-a form, opening a modal, canceling an action, or performing an add or delete
-operation. Almost all interactions in the Carbon Design System are implemented
-using the native `<button>` element.
+Carbon bakes keyboard operation into its components, improving the experience of blind users and others who operate via the keyboard. Carbon incorporates many other accessibility considerations, some of which are described below.
 
-By default, interactive elements like `<button>` are included in the tab order
-of the document. When a button has focus, both the `Space` and `Enter` keys will
-activate the button. If the button is disabled, then the `disabled` attribute is
-provided.
+### Keyboard interactions
 
-The label for a button can be computed by the contents of the underlying
-`<button>`, or provided by the `aria-label` or `aria-labelledby` attributes.
-Additional information about the button may be supplied by the
-`aria-describedby` attribute.
+<Row>
+<Column colLg={8}>
 
-## Accessibility considerations
+![example of button keyboard interaction](images/button-accessibility-1.png)
 
-This component has been validated to meet the
-[WCAG 2.1 AA](https://www.w3.org/TR/WCAG21/) and
-[Section 508](http://www.section508.gov/) accessibility guidelines, however
-changes made by the content owner can affect accessibility compliance. Be sure
-to review and follow the guidance in this section when updating or adding new
-content to this component.
+<Caption>Buttons can be reached by `Tab` and selected with `Space` or `Enter`.</Caption>
 
-1. The button can be activated using both the `Space` or `Enter` key.
-1. The button should have text or label that accurately describes the action of
-   the button.
-1. If a description of the button's function is present, the button element
-   should have an `aria-describedby` set to the ID of the element containing the
-   description.
-1. After the button is activated ensure the focus is set correctly based on the
-   type of action the button performs. See
-   [W3C WAI-ARIA Authoring Practices Button Design Pattern](https://www.w3.org/TR/wai-aria-practices-1.1/#button)
-   for a list of examples.
+</Column>
+</Row>
 
-## Resources
+### interaction
```

Comment:
Capital `I` and add a `s` since there are multiple 
```suggestion
### Interactions
```

Reply:
I actually changed it to Behavior

Reply:
I actually changed it to Behavior